thevoid/connection.cpp

```diff
@@ -796,6 +873,15 @@ void connection<T>::process_data()
 				send_error(http_response::bad_request);
 				return;
 			} else {
+				CONNECTION_INFO(
+					"received new request: method: %s, url: %s, local: %s, remote: %s, headers: %s",
```

Comment:
Good point.

It looks like NGINX has something similar, one can use `http_<header>` and `sent_http_<header>` variables to log request and response headers:
http://nginx.org/en/docs/http/ngx_http_core_module.html#variables
http://nginx.org/en/docs/http/ngx_http_log_module.html

I'll attempt to add list of headers to log as a config parameter.